### DISCUSSION TITLE
No-ip support

### DISCUSSION THREAD

**Discussion by pitsi** [troglobit/inadyn#395] [Q&A]
I am on debian unstable, I have a free subscription on no-ip and I use inadyn as a fallback to my router's built-in function for dynamic dns.

From 2013 to 2021 debian's inadyn package was stuck to v1.99.x, because its maintainer probably gave up on it and officially orphaned it on 2017. That version did support no-ip.
Last October (2021), a new maintainer updated the package to 2.8.1 and just yesterday he updated it to 2.9.1. To be honest, I had not realised that the config file had changed for v2.x until today that I checked it. And I see no lines for no-ip or even an example config under /usr/share/doc/inadyn/examples/.

So, is no-ip still supported by inadyn?

**Comment by troglobit** [comment]
First of all, questions regarding packaging of In-a-Dyn should go to your distribution, in this case Debian.  I have no control over what files and information they add to packages.

no-ip.com is supported, both the [README](https://github.com/troglobit/inadyn/blob/master/README.md) and the manual page [inadyn.conf(5)](https://manpages.debian.org/unstable/inadyn/inadyn.conf.5.en.html) document this.

**Comment by pitsi** [comment]
You are right, I should have checked either the readme or the man page.
However, since I do not have that complex "multiple usernames" scenario that the readme and the manpage cover, I would like to ask if this is enough
```
provider no-ip.com {
    username   = myusername
    password   = mypassword
    hostname   = myhostname.no-ip.com
}
```
I have disabled my router's dynamic dns built-in function so as to check.

As for debian, I know that its huge delay in packaging inadyn is not your fault and I was among the ones that requested an update to the package after all those years.

**Reply by troglobit** [reply]
Yes, that should be enough.

**Reply by pitsi** [reply]
Thank you :)
